lms/static/js/groups/views/cohort_discussions_course_wide.js

```diff
@@ -33,8 +33,10 @@
                             entries = courseWideDiscussions.entries,
                             children = courseWideDiscussions.children;
 
-                        return HtmlUtils.joinHtml.apply(this, _.map(children, function(name) {
-                            var entry = entries[name];
+                        return HtmlUtils.joinHtml.apply(this, _.map(children, function(child) {
+                            // child[0] is the category name, child[1] is the type (i.e. 'entry' or 'subcategory')
+                            var name = child[0],
+                                entry = entries[name];
                             return subCategoryTemplate({
```

Comment:
There's something odd about the names used here.  Once you help clarify, it would be great to either fix the names and/or add a comment that provides clarity.

In the other loop over children (see common/static/common/js/discussion/views/discussion_topic_menu_view.js), we use entryTemplate() for entries and categoryTemplate() for subcategories.  Here, it looks like these are entries because you are pulling from an `entries` list, but then you are using a `subCategoryTemplate`.  Can you explain explain this?

Additionally, are all of these children guaranteed to be entries?  If so, you should add a comment stating so, and possibly clarify why.  If not, then type should be used and you should determine what needs to be done if child[1] is `'subcategory'`.